server/boot/migrate-builid-in-model.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function(app) {
+  let lbTables = ['User', 'AccessToken', 'ACL', 'RoleMapping', 'Role', 'CustomUser'];
```

Comment:
could we use db-migrate? migration should be depend on our decision, it always runs when starting server now